### DISCUSSION TITLE
Issues after using pushkin armageddon

### DISCUSSION THREAD

**Discussion by MelindaPozzi** [l3atbc-datadog/pushkin#138] [General] [answered]
Hi!

I updated my pushkin version from 1.5.5 to 2.0.0 (thinking that maybe this was the reason why I could not connect to the local host). But then I had some issues and I first tried _pushkin kill_ and then _pushkin armageddon_.

Now I have this error message when I run _pushkin prep_:

![Error1](https://user-images.githubusercontent.com/33163405/90527778-37635700-e172-11ea-9f83-fa076af8a4a5.JPG)

And I have this question that I cannot answer when I run _pushkin start_:

![Error2](https://user-images.githubusercontent.com/33163405/90527818-40ecbf00-e172-11ea-817a-a29b37a9f5f5.JPG)

Thank you in advance for your help!
Mélinda

**Comment by parkerrobbins** [comment]
Hi, @MelindaPozzi! A couple of other users have reported similar issues (e.g., #88, #100, #135). Your issue appears to be slightly different. 

1. Could you double-check that the Pushkin CLI did indeed update to 2.0.0 by running `pushkin --version`? Some users had trouble upgrading to the newest version (e.g., #132, #134).
2. Could you check that Docker is configured correctly by running `docker run hello-world`?
3. You may need to reinstall the site and experiments in a new directory (you can save your current directory and re-add any customizations you may have made to the code in the new directory). That worked for another user who had trouble after the update.

You are running this on Windows via an AWS instance, right?

**Reply by MelindaPozzi** [reply]
Hi!

Sorry for my late answer. Yes, I'm running this on Windows via an AWS instance.

I have the newest version (2.0.0) of Pushkin CLI and Docker seems to be configured correctly:

![Pushkin issue](https://user-images.githubusercontent.com/33163405/91562486-9036aa80-e93d-11ea-8842-e074fd298627.JPG)

I've tried to create a new directory but when I run _pushkin prep_, I got this error message:

![Pushkin issue 2](https://user-images.githubusercontent.com/33163405/91593595-c0437500-e960-11ea-8a0f-e2de1e6a715a.JPG)

Thank you for your help!
Mélinda

**Reply by parkerrobbins** [reply]
Hi, @MelindaPozzi! 

1. The newest version is now 2.2.10. You can update by running: `yarn global upgrade pushkin-cli --latest`.
2. After updating, please run `pushkin armageddon` again.
3. If you have customized your Pushkin site or experiment(s), then follow [these instructions](http://www.movingresearchonline.info/session_resources/pushkin_migration.html), which involve [this pushkin.yaml file](http://www.movingresearchonline.info/session_resources/pushkin.yaml) and [this config.js file](http://www.movingresearchonline.info/session_resources/config.js) to migrate your existing site to work with the current version of the CLI.
4. If you were just making a test (no real customizations), then I would recommend just creating a new pushkin site. Be sure to use the `basic` site template, not the `default` site template, as the `basic` site template is the one that's going to allow you to test on Windows using your AWS EC2 instance.

Please let me know if you have any questions or need any more help.

**Comment by MelindaPozzi** [comment, marked as answer]
Hi!

Thank your for your answer! When I try to connect from Putty to my AWS EC2 instance, I got this error message:

![Pushkin issue 3](https://user-images.githubusercontent.com/33163405/91600718-bcb3ec00-e968-11ea-9e03-77c7387ee088.JPG)

So I can't even update Pushkin CLI.

**Reply by parkerrobbins** [reply]
So it looks like you don't have any space remaining on your instance. You can find information [here](https://aws.amazon.com/premiumsupport/knowledge-center/expand-root-ebs-linux/) about how to increase the amount of disk space on your EC2 instance.

**Reply by parkerrobbins** [reply]
You might find [this tutorial](https://medium.com/findworka/how-to-increase-disk-size-for-an-ec2-instance-on-aws-b82181df6215) a bit easier than reading the AWS docs.

**Reply by jkhartshorne** [reply]
@parkerrobbins - since this has come up a few times, do you think it makes sense to add something to the FAQ?

**Reply by parkerrobbins** [reply]
@jkhartshorne Definitely. I'll add an issue.

**Comment by MelindaPozzi** [comment]
Hi again!

I've followed all your instructions and it finally works!

![pushkin site](https://user-images.githubusercontent.com/33163405/91634723-cbdf7c00-e9f2-11ea-8e48-4562c0d60000.JPG)

Thank you very much for your help!
Mélinda

**Reply by jkhartshorne** [reply]
Awesome!

**Comment by restmegan** [comment]
Hi @MelindaPozzi ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!

**Reply by MelindaPozzi** [reply]
Hi @restmegan,

Thank you for asking ! I could solve all the issues thanks to the team's answers, so everything is fine for me.

A big thank you to all the team for this worshop!
Mélinda
